hyperspy/tests/utils/test_map.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+
+from hyperspy.signal import BaseSignal as hssig
+from hyperspy.signals import Signal1D as hsspc
+from hyperspy.signals import Signal2D as hsimg
+from scipy.ndimage import gaussian_filter as gf
+from itertools import cycle
+
+class TestMapScalar:
+    """
+    Test map onto signal with scalar parameter(s). Proceed by applying a
+    Gaussian filter to all the images in an image dataset using variable sigma
+    parameter(s)
+    """
+    def setup_method(self, method):
+        self.sigma  = 2.5
+        self.sigmas = hssig(np.linspace(2,5,10)).T
+        self.image  = hsimg(np.random.random((10, 64, 64)))
+        self.image.metadata.General.title = 'test'
+
+    def test_A(self):
```

Comment:
Better test names should be given - when the test fails, you should aim for it to be immediately clear what was broken. `test_A` only makes sense if you either know exactly what it does, or open the test to begin with, and the goal is to avoid having to do so.